src/Avalonia.Controls/Utils/BorderRenderHelper.cs

```diff
@@ -15,8 +17,13 @@ internal class BorderRenderHelper
         private Size _size;
         private Thickness _borderThickness;
         private CornerRadius _cornerRadius;
+        private AvaloniaList<double> _borderDashArray;
```

Comment:
I'm seeing a warning that these fields are unused on `master` - should they have been removed?

Reply:
Y they are not used anymore

Reply:
Yes, this is my mistake, I didn't notice. But this is great news, as it means that the problem of new properties has been solved :)

now I will do a PR with a fix